process/types.go

```diff
@@ -119,6 +119,13 @@ type Process interface {
 	// CalculateMappingFileID calculates FileID of the backing file
 	CalculateMappingFileID(*Mapping) (libpf.FileID, error)
 
+	// ExtractAsFile returns a filename suitable for opening the given file from
+	// the target process namespace. A last resort method to access the file if
```

Comment:
```suggestion
	// the target process namespace. It's a last resort method to access the file if
```

Reply:
Reworded the message to be a bit more descriptive.